components/pd_client/src/util.rs

```diff
@@ -28,41 +26,14 @@ use tikv_util::security::SecurityManager;
 use tikv_util::timer::GLOBAL_TIMER_HANDLE;
 use tikv_util::{Either, HandyRwLock};
 
-pub struct PdClientStub {
-    pd_client: PdClient,
-    config_client: ConfigClient,
-}
-
-impl PdClientStub {
-    fn new(ch: Channel) -> Self {
-        // reuse the same channel for tow different client
-        PdClientStub {
-            pd_client: PdClient::new(ch.clone()),
-            config_client: ConfigClient::new(ch),
-        }
-    }
-
-    pub fn config(&self) -> &ConfigClient {
-        &self.config_client
-    }
-}
-
-impl Deref for PdClientStub {
-    type Target = PdClient;
-
-    fn deref(&self) -> &Self::Target {
-        &self.pd_client
-    }
-}
-
 pub struct Inner {
     env: Arc<Environment>,
     pub hb_sender: Either<
         Option<ClientDuplexSender<RegionHeartbeatRequest>>,
         UnboundedSender<RegionHeartbeatRequest>,
     >,
     pub hb_receiver: Either<Option<ClientDuplexReceiver<RegionHeartbeatResponse>>, Task>,
-    pub client_stub: PdClientStub,
+    pub client_stub: PdClient,
```

Comment:
#5594 renames the `PdClient` to `PdClientStub`, should we keep the name?

Reply:
Sorry I miss that, I will change it back.